modules/smtp/site.js

```diff
@@ -327,13 +327,14 @@ var text_to_bubbles = function(input) {
     var contact_type = input.getAttribute("data-type");
     var contact_source = input.getAttribute("data-source");
 
-    if ($(input).val() && contact_id) {
+    if ($(input).val()) {
         var recipients = $(input).val().split(/,|;/);
         var invalid_recipients = '';
 
         for (var i = 0; i < recipients.length; i++) {
             if (is_valid_recipient(recipients[i])) {
-                append_bubble(recipients[i].trim(), input, contact_id, contact_type, contact_source);
+                const value = recipients[i].trim().replace(/[<>]/g, '')
```

Comment:
Why do we remove the brackets before adding the bubble? `Name <email>` is a valid format of recipients according to the RFC.

Reply:
Yes, it is, but when we display the bubble we don't show those brackets. The format that we display is `Name email`.

Reply:
I don't think we should strip the brackets in that case. It is standard display in email clients to display the brackets, so unless there is a compelling reason not to display them, can you return them back?

Reply:
Alright, let's go for it.